NuGet.config

```
@@ -9,7 +9,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
-    <add key="darc-pub-dotnet-emsdk-9506882" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-95068821/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-44da825" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-44da825c/nuget/v3/index.json" />
```

Re-adding. This keeps causing CI failures.
```suggestion
    <add key="darc-pub-dotnet-emsdk-9506882" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-95068821/nuget/v3/index.json" />
    <add key="darc-pub-dotnet-emsdk-44da825" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-44da825c/nuget/v3/index.json" />
```